__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const app = require("../app.js");
+const request = require("supertest");
+const db = require("../db/connection.js");
+const seed = require("../db/seeds/seed.js");
+const data = require("../db/data/test-data");
+
+beforeEach(() => {
+    return seed(data);
+})
+
+afterAll(() => {
+    return db.end();
+})
+
+
+describe('GET /api/topics', () => {
+    test('returns 200 status code when request is successfully fullfilled. ', () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+    });
+
+    test('receive array of 3 topic objects and each object to have slug & description strings', () => {
+        return request(app)
+            .get("/api/topics")
+            .then(({ body }) => {
+                expect(body.topics).toHaveLength(3);
```

Comment:
Great test, love that you've checked the length to make sure the forEach loop isn't giving us false positives 🙌